phoenix/slackbot/views.py

```diff
@@ -342,6 +342,14 @@ def announce(request):
                     "hint": """If sales have been affected choose Postmortem report
                                                   in Suggested outcome.""",
                 },
+                {
+                    "label": "B2B Partners affected",
+                    "type": "select",
+                    "options": SALES_AFFECTED_CHOICE_OPT,
+                    "name": "b2b_partners_affected_choice",
+                    "hint": """If b2b partners have been affected choose Postmortem report
+                                                  in Suggested outcome.""",
```

Comment:
Should probably be on online or have better formatting.

Reply:
I just copied the formatting of the previous message, not sure if it has some impact, so I'd rather leave it like this 😆